specification/datalake-store/resource-manager/Microsoft.DataLakeStore/stable/2016-11-01/examples/VirtualNetworkOrSubnets_DeleteById.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
How do you write this example? Does the example come from real test case?

Reply:
added a fake guid